### DISCUSSION TITLE
How to show the h2 and the h1 on the same slide ?

### DISCUSSION THREAD

**Discussion by ddotta** [quarto-dev/quarto-cli#7008] [Q&A] [answered]
### Description

In my quarto presentation (revealjs), is it possible to add the h1 title on which the h2 title of my slide depends?

![image](https://github.com/quarto-dev/quarto-cli/assets/13397519/fc7ec2ae-5151-431f-99c2-afef61a5cb5b)

My code looks like this :  

````qmd
# Styles par défaut {.backgroundTitre}

## Style du texte {.backgroundStandard}

#### En markdown
````

**Comment by cderv** [comment]
Hello @ddotta 

can you Edit the title of this question ? Thanks 

also, do I understand correctly that you want to show on a slide also the h1 title part ?

**Comment by ddotta** [comment]
Sorry @cderv !
It's done, I'd like to add it automatically without having to remind it of the h1 on which the h2 depends.

**Reply by cderv** [reply, marked as answer]
Same answer as the other question, you could probably do what you want using Lua filter to catch H1 header content, and then show it the way you want on another slide. 

Maybe there is an extension existing that could help you. I know this one but a bit different: https://github.com/andrie/reveal-auto-agenda

**Reply by ddotta** [reply]
Thanks @cderv I dont know if I will be able to create this lua filter... we'll see

**Comment by mcanouil** [comment]
FYI, there is a "breadcrumb" plugin for Reveal.js: https://github.com/sedrubal/reveal.js-breadcrumb

**Reply by ddotta** [reply]
Thanks @mcanouil !

**Comment by ddotta** [comment]
FYI, I was able to do what I wanted thanks to [this](https://github.com/shafayetShafee/reveal-header)
